build/build.sh

```diff
@@ -28,11 +32,14 @@ pandoc --verbose \
 
 # Create PDF outpout
 echo "Exporting PDF manuscript"
+cd output
```

Comment:
There may be a way around this, but the PDF is already being built in a different location than `sections/` (and `sections/images`) so any path used to reference the images in the Markdown will also have to work for the PDFs. That is, if the Markdown file references `.images/image.png` then the PDF converter must also be able to find that. Any thoughts?